subcommands/targets/tag.go

```diff
@@ -22,21 +22,21 @@ var (
 func init() {
 	var tagCmd = &cobra.Command{
 		Use:   "tag <target> [<target>...]",
-		Short: "Apply a comma separated list of tags to one or more targets.",
+		Short: "Apply a comma separated list of tags to one or more Targets.",
 		Example: `
-  # Promote Target #42 currently tagged as master
-  fioctl targets tag --tags master,promoted --by-version 42
+  # Promote Target #42 currently tagged as main
+  fioctl targets tag --tags main,promoted --by-version 42
 
   # Tag a specific Target by name
-  fioctl targets tag --tags master,testing intel-corei7-64-lmp-42`,
+  fioctl targets tag --tags main,testing intel-corei7-64-lmp-42`,
```

Comment:
IIUC these examples are based off the real-life user repositories.
Did we already switch git.foundries.io to use `main` as a default branch for all users' Factories?

Reply:
@doanac @mike-sul Do you know the answer?